sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -129,17 +130,17 @@ public HttpPipeline getHttpPipeline() {
      *
      * @return the customer provided key used for encryption.
      */
-    public CpkInfo getCustomerProvidedKey() {
-        return client.getCustomerProvidedKey();
+    CpkInfo getCustomerProvidedKey() {
```

Comment:
Why did we make this package private instead of removing it, and same goes for `getServiceVersion`, as you did in `BlobAsyncClientBase`?

Reply:
Might a miss on clean up